src/PresentationalComponents/RuleDetails/_RuleDetails.scss

```diff
@@ -0,0 +1,7 @@
+.pf-l-grid.actions__detail .pf-l-grid__item.pf-u-pb-md  {
+    border-bottom: .01rem solid #ccc;
+}
+.ins-sev-clr-1 { color: var(--pf-global--Color--200); }
+.ins-sev-clr-2 { color: var(--pf-global--warning-color--100); }
+.ins-sev-clr-3 { color: #ec7a08; }
```

Comment:
no pf-global color for this? 😟

Reply:
Unfortunately no pf-global, but I found a `--ins-color--orange` in ifc and am using that (same color).  Will update the Shield component to use it as well to ensure consistency.